app/build.gradle

```diff
@@ -125,6 +125,7 @@ android {
         buildConfigField "Float", "DEFAULT_WINDOW_DISTANCE", "0.0f"
         buildConfigField "Boolean", "ENABLE_PAGE_ZOOM", "false"
         buildConfigField "Boolean", "USE_SOUNDPOOL", "true"
+        buildConfigField 'String', 'EXPERIENCES_ENDPOINT', '"https://darker.ink/wolvic/experiences.json"'
```

Comment:
I guess we should use an Igalia URL here

Reply:
Yes, that's just for testing purposes. Eventually this should point to the same repository as our `props.json`.

The file is in my Wolvic fork: https://github.com/felipeerias/wolvic/blob/gh-pages/experiences.json